apps/tufctl/main.cpp

```diff
@@ -0,0 +1,50 @@
+#include "aktualizr-lite/tuf/aklite-tuf.h"
```

Comment:
It think it doesn't make sense to name the header file as `aklite-tuf.h` since it's already under "aktualizr-lite" namespace/prefix. I would call it just `include/aktualizr-lite/tuf/tuf.h`